src/module.rs

```diff
@@ -321,59 +321,56 @@ impl<'ctx> Module<'ctx> {
         Some(BasicTypeEnum::new(type_))
     }
 
-    /// Sets a `Target` to this `Module`.
+    /// Assigns a `TargetTriple` to this `Module`.
     ///
     /// # Example
     ///
     /// ```rust,no_run
     /// use inkwell::context::Context;
-    /// use inkwell::targets::Target;
-    ///
-    /// Target::initialize_x86(&Default::default());
+    /// use inkwell::targets::TargetTriple;
     ///
     /// let context = Context::create();
     /// let module = context.create_module("mod");
-    /// let target = Target::from_name("x86-64").unwrap();
+    /// let triple = TargetTriple::create("x86_64-pc-linux-gnu");
     ///
-    /// assert!(module.get_target().is_none());
+    /// assert_eq!(module.get_target(), TargetTriple::create(""));
     ///
-    /// module.set_target(&target);
+    /// module.set_target(&triple);
     ///
-    /// assert_eq!(module.get_target().unwrap(), target);
+    /// assert_eq!(module.get_target(), triple);
     /// ```
-    pub fn set_target(&self, target: &Target) {
+    pub fn set_target(&self, target_triple: &TargetTriple) {
```

Comment:
We should change these to `get/set_target_triple` for clarity, despite LLVM's poorly named function that implies an actual Triple object

Reply:
Apparently that's the name the corresponding function in the [C++ API](https://llvm.org/doxygen/classllvm_1_1Module.html#af66bad1510da8a71d20002609d3b4a25) has

Reply:
I called it `get/set_triple` for consistency with the rest of the `inkwell` api.